docs/src/tutorials/supported_operations.md

```diff
@@ -29,6 +35,8 @@ As per specification the configuration file needs to be a `toml` file which desc
 
 `thin-edge.io` stores the operations configuration files in the `/etc/tedge/operations/<cloud-provider>/` directory.
 
+`thin-edge.io` stores the operations configuration files for the child devices in the `/etc/tedge/operations/<cloud-provider>/<child-device>` directory.
```

Comment:
These two sentences are too repetitive. Make a single sentence that introduces the 2 levels.

You can move here the text related to child-device boot strap.

Reply:
moved child device-related things into the bootstrap part.